docker-compose.yml

```diff
@@ -19,6 +19,46 @@ services:
       - POSTGRES_HOST_AUTH_METHOD=trust
       - POSTGRES_DB=ferretdb
 
+  # Tigris configuration is based on https://github.com/tigrisdata/tigris/blob/main/test/docker/docker-compose.yml
+
+  tigris_fdb:
+    build:
+      context: ./build/deps
+      dockerfile: tigris_fdb.Dockerfile
+    container_name: ferretdb_tigris_fdb
+    ports:
+      - 4500:4500
+    volumes:
+      - tigrisdata:/var/fdb/
+      # - tigrisdata:/var/lib/foundationdb
+
+  # tigris_typesense:
+  #   build:
+  #     context: ./build/deps
+  #     dockerfile: tigris_typesense.Dockerfile
+  #   container_name: ferretdb_tigris_typesense
+  #   environment:
+  #     - TYPESENSE_API_KEY=ts_test_key
+  #     - TYPESENSE_DATA_DIR=/tmp
+  #   ports:
+  #     - 8108:8108
+
+  tigris:
+    build:
+      context: ./build/deps
+      dockerfile: tigris.Dockerfile
+    container_name: ferretdb_tigris
+    depends_on: [tigris_fdb]
+    restart: on-failure
+    command: bash -c 'fdbcli --exec "configure new single memory" && /server/service'
+    environment:
+      - TIGRIS_SERVER_SEARCH_AUTH_KEY=ts_test_key
+      - TIGRIS_SERVER_SEARCH_HOST=tigris_typesense
+    ports:
+      - 8081:8081
+    volumes:
+      - tigrisdata:/etc/foundationdb
```

Comment:
This part is unclear to me. There are mentions of `foundationdb` commented in some files, but it's uncommented here. Why?

Reply:
The next version of Tigris will change this path